cluster-autoscaler/utils/taints/taints.go

```diff
@@ -108,6 +117,23 @@ var (
 	// Mutable only in unit tests
 	maxRetryDeadline      time.Duration = 5 * time.Second
 	conflictRetryInterval time.Duration = 750 * time.Millisecond
+
+	explicitlyReportedNodeTaints = TaintKeySet{
```

Comment:
This is duplicating the list from `NodeConditionTaints`, can we initialize this with `NodeConditionTaints` with a few extra taints? I worry they may get out of sync.

Reply:
Changed, PTAL